pkg/image/image_test.go

```diff
@@ -72,3 +72,154 @@ func TestEdgeSet(t *testing.T) {
 		})
 	}
 }
+
+func TestCloudSet(t *testing.T) {
+	type args struct {
+		imageRepository string
+		version         string
+	}
+	tests := []struct {
+		name string
+		args args
+		want Set
+	}{
+		{
+			name: "repo nil, ver not nil",
+			args: args{
+				imageRepository: "",
+				version:         "v1.12.0",
+			},
+			want: Set{
+				CloudAdmission:         "kubeedge/admission:v1.12.0",
+				CloudCloudcore:         "kubeedge/cloudcore:v1.12.0",
+				CloudIptablesManager:   "kubeedge/iptables-manager:v1.12.0",
+				CloudControllerManager: "kubeedge/controller-manager:v1.12.0",
+			},
+		},
+		{
+			name: "repo nil, ver nil",
+			args: args{
+				imageRepository: "",
+				version:         "",
+			},
+			want: Set{
+				CloudAdmission:         "kubeedge/admission",
+				CloudCloudcore:         "kubeedge/cloudcore",
+				CloudIptablesManager:   "kubeedge/iptables-manager",
+				CloudControllerManager: "kubeedge/controller-manager",
+			},
+		},
+		{
+			name: "repo not nil, ver not nil",
+			args: args{
+				imageRepository: "kubeedge-test",
+				version:         "v1.12.0",
+			},
+			want: Set{
+				CloudAdmission:         "kubeedge-test/admission:v1.12.0",
+				CloudCloudcore:         "kubeedge-test/cloudcore:v1.12.0",
+				CloudIptablesManager:   "kubeedge-test/iptables-manager:v1.12.0",
+				CloudControllerManager: "kubeedge-test/controller-manager:v1.12.0",
+			},
+		},
+		{
+			name: "repo not nil, ver nil",
+			args: args{
+				imageRepository: "kubeedge-test",
+				version:         "",
+			},
+			want: Set{
+				CloudAdmission:         "kubeedge-test/admission",
+				CloudCloudcore:         "kubeedge-test/cloudcore",
+				CloudIptablesManager:   "kubeedge-test/iptables-manager",
+				CloudControllerManager: "kubeedge-test/controller-manager",
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			if got := CloudSet(tt.args.imageRepository, tt.args.version); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("CloudSet() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestSet_Get(t *testing.T) {
+	tests := []struct {
+		name string
+		s    Set
+		args string
+		want string
+	}{
+		{
+			name: "get image version",
+			s:    Set{"kubeedge-test/cloudcore": "1.12.0"},
+			args: "kubeedge-test/cloudcore",
+			want: "1.12.0",
+		},
```

Comment:
```suggestion
		{
			name: "get cloudcore image",
			s:    Set{"cloudcore": "kubeedge-test/cloudcore:1.12.0"},
			args: "cloudcore",
			want: "kubeedge-test/cloudcore:1.12.0",
		},
```
This case needs to be modified, everything else is ok.

Reply:
Adjusted, thanks